### DISCUSSION TITLE
CSS being included twice

### DISCUSSION THREAD

**Discussion by praveshishere** [nuxt/nuxt#16849] [Questions]
We are using Tailwind CSS (Nuxt tailwind plugin) in our project for styling, but for some reason, styles are being included twice after upgrading to RC-10

The sources are:
- All my CSS inlined into the HTML
- Also, entry.css is being fetched and parsed

So, in my thought RC-10 included a feature to inline critical CSS in the HTML response itself. which it does but includes all of the compiled CSS, which is fine with me, but it makes a request to the original entry.css file which seems unnecessary.

Is there a way to tell nuxt not to fetch entry.css as we already have all the styles?

**Comment by danielroe** [comment]
Vite is fetching `entry.css` (we're discussing how best to avoid this) but is doing so _later_ in the waterfall.

Let's track in https://github.com/nuxt/framework/issues/7619.
